### PR TITLE
Add hwb, lab, and lch to functions matching isColor

### PR DESCRIPTION
**Pull request by jonathantneal**
This PR contains:

- [ ] bugfix
- [x] feature
- [ ] refactor
- [x] tests
- [ ] documentation
- [ ] metadata

### Breaking Changes?

- [ ] yes
- [x] no

---

This PR changes the way the `isColor` property of a `Func` is determined so that it may include [`hwb()`](https://drafts.csswg.org/css-color/#the-hwb-notation), [`lch()`](https://drafts.csswg.org/css-color/#funcdef-lch), and [`lab()`](https://drafts.csswg.org/css-color/#funcdef-lab) notations.

---

_Note: This is not causing me any issues in my current work. It’s just something I noticed while inspecting the code._

_Note: I would expect the `isColor` and `isVar` helper properties to be getters, so that in the event one reassigns a function name, they remain accurate. I am open to adding that in a separate PR if you feel it will add value._

### CODE REVIEW

File: lib/nodes/Func.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const { registerWalker } = require('../walker');
 
 const Container = require('./Container');
 
-const colorFunctions = ['hsl', 'hsla', 'rgb', 'rgba'];
+const colorFunctionsLowerRegex = /^(hsla?|hwb|lab|lch|rgba?)$/;
```
Comment (shellscape) [inline lib/nodes/Func.js:15]:
```suggestion
const reColorFunctions = /^(hsla?|hwb|lab|lch|rgba?)$/i;
```

It looks like units are case insensitive https://www.w3.org/TR/css-values-3/#dimensions, so we can modify the regex accordingly and also do away with the `lowerName` on line 106 if we want to.

**Comment by shellscape** [comment]
I checked out CI and it looks like a few snapshots need to be update, but please do take a second look.

**Comment by jonathantneal** [comment]
I GitHub goofed, so the fixes you’ve requested have been moved into https://github.com/shellscape/postcss-values-parser/pull/111

File: lib/nodes/Func.js (hunk 1 above)
Comment (jonathantneal) [reply lib/nodes/Func.js:15]: Done in https://github.com/shellscape/postcss-values-parser/pull/111/commits/b95c4c8bcdd541be6aa93da16ac0e51ab5188549

**Pull request closed without merging**